proto/server/BUILD

```diff
@@ -0,0 +1,20 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
```

Comment:
Can you put these files in example/usps_api instead?

Reply:
Done